scripts/configure-installer-tests-cluster.sh

```diff
@@ -13,8 +13,13 @@ KUBEADMIN_PASSWORD_FILE=${KUBEADMIN_PASSWORD_FILE:-"${DEFAULT_INSTALLER_ASSETS_D
 # Default values
 OC_STABLE_LOGIN="false"
 CI_OPERATOR_HUB_PROJECT="ci-operator-hub-project"
+# Copy kubeconfig to temporary kubeconfig file
+# Read and Write permission to temporary kubeconfig file
+mkdir -p ${DEFAULT_INSTALLER_ASSETS_DIR}/tmp
```

Comment:
Instead of tmp can we name the dir what it stand for. For example ```kubeconfig```

Reply:
I would go with tmp because its not the original `kubeconfig`, someone might confuse with this. As we are copying the `kubeconfig` and using it for the particular test, IMO keeping this under `tmp` make more sense than in dir like `kubeconfig`.